src/Storage/Commands.Storage/Common/Cmdlet/NewAzureStorageContext.cs

```diff
@@ -69,6 +72,18 @@ public class NewAzureStorageContext : AzureDataCmdlet
         /// </summary>
         private const string AnonymousParameterSet = "AnonymousAccount";
 
+        /// <summary>
+        /// OAuth storage account parameter set name
+        /// </summary>
+        private const string OAuthParameterSet = "OAuthAccount";
+
+        /// <summary>
+        /// OAuth storage account parameter set name
+        /// </summary>
+        private const string OAuthEnvironmentParameterSet = "OAuthAccountEnvironment";
+
+        private IAccessToken accessToken = null;
```

Comment:
does this belong on the cmdlet?  seems like you should have this in the StorageContext, or use a closure to capture the value

Reply:
removed, use a closure.